ql/src/java/org/apache/hadoop/hive/ql/ddl/table/info/show/tables/ShowTablesAnalyzer.java

```diff
@@ -53,17 +56,29 @@ public void analyzeInternal(ASTNode root) throws SemanticException {
     String tableNames = null;
     TableType tableTypeFilter = null;
     boolean isExtended = false;
```

Comment:
nit: better to use `isIceberg`, to have uniform variable naming as above line `isExtended`